src/pages/auth/login.tsx

```diff
@@ -1,9 +1,19 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { useNavigate } from "react-router-dom";
+import { useAuth } from "../../data/authentication";
+import { useCookies } from "react-cookie";
 
 export const Login: React.FC = () => {
   const [user, setUser] = useState({ email: "", password: "" });
   const navigate = useNavigate();
+  const { login } = useAuth();
```

Comment:
To avoid repeating the redirection you could create empty wrapper to only manage the logic of redirection on login and register, we need to reduce redundant code